### PR TITLE
julia-backend First Pull

### PR DESCRIPTION
**Pull request by juliawoodcock**
Pls

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Current DB status:
```
Comment (draymond63) [inline backend/README.md]: I'd prefer to include this as a section of the top level readme

File: backend/app.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# - sensor sends data extracted from yaml file to backend when it gets a series of steps
+# - backend reads the yaml file for sensorId, floor, fs, user, ts data
+# - ts data goes in a table with timestamps, ts data at timestamps, and sensorId. 
+# - seperate sensorId table stores the floor, fs, user, ts data information
+
+
+# Hello World (Daniel)
+@app.route('/')
+def hello_world():
+    return 'Hello, Daniel!'
```
Comment (draymond63) [inline backend/app.py:89]: HI :)

File: backend/backend_requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+flask
```
Comment (draymond63) [inline backend/backend_requirements.txt]: We should put version #s for all dependencies

File: backend/app.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            _id=data['sensorid'],
+            sampling=data['sampling'],
+            floor=data['floor'],
+            user=data['user']
+        )
+
+        db.session.add(new_data)
+        db.session.commit()
+
+        return jsonify({"message": "Data added successfully"}), 201
```
Comment (draymond63) [inline backend/app.py]: These http codes shouldn't be magic constants, they should be defined as a constant/variable. i.e. `HTTP_CREATED = 201`

File: backend/test.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import requests
```
Comment (draymond63) [inline backend/test.py]: Test is a bad name for a file. Maybe `client.py`?

File: requirements.txt [hunk 6]
```diff
@@ -1,3 +1,3 @@
 numpy>=1.26.1
 plotly>=5.17.0
-pyserial>=3.5
+pyserial>=3.5
```
Comment (draymond63) [inline requirements.txt]: Why was this changed?

**Review by draymond63 — COMMENTED** [review]
Great stuff julia! Just some design comments, but I'm happy to merge this and clean up some of the stuff in a new PR with more feature work :)

**Review by draymond63 — APPROVED** [review]

**Comment by juliawoodcock** [comment]
@draymond63 pushed some changes per your comments :)
